README.md

```diff
@@ -157,8 +152,7 @@ takes a list of modules and returns a `patch` function that uses the
 specified set of modules.
 
 ```mjs
-import { classModule } from 'snabbdom/modules/class'
-import { styleModule } from 'snabbdom/modules/style'
+import { classModule, styleModule } from 'snabbdom'
```

Comment:
In the interests of making all of the example code sections smaller and more readable, I think we can omit all of the `import` statements _after_ the first example. We don't need to demo the import process in every example snippet.